lambda/src/server/main.cljs

```diff
@@ -33,16 +32,16 @@
   (p/let [[kind data props] (cond (:payload body) [:interaction
                                                    (:payload body)
                                                    #:slack{:team-id (-> body :payload :team :id)
-                                                           :user-id (-> body :payload :user :id)
-                                                           :app-id (-> body :payload :api_app_id)}]
+                                                           :user-id (-> body :payload :user :id)}]
                                   (:event body) [:event
                                                  (:event body)
                                                  #:slack{:team-id (:team_id body)
                                                          :user-id (-> body :event :user)
                                                          :app-id (:api_app_id body)}]
                                   (:challenge body) [:challenge (:challenge body) nil])
-          token (when (:slack/team-id props)
-                  (slack-db/team->token props))
+          token (if (and (:slack/team-id props) (:slack/app-id props))
```

Comment:
am thinking we should stop including `app-id` in `props` entirely, and also not back-off to the hard-coded token here, to avoid unexpected surprises in prod

Reply:
Fixed in eb9375a